### DISCUSSION TITLE
希望后续可以实现的功能

### DISCUSSION THREAD

**Discussion by chengzhongxue** [halo-dev/halo#4750] [Ideas]
### Your current Halo version

2.10

### Describe this feature

1. 后续会实现类似于wp哪些主题那样的论坛功能？
![image](https://github.com/halo-dev/halo/assets/89380218/a3a15b38-1bec-44a7-a40c-c026f9beb0a0)

2. 后续会实现商城？

**Comment by ruibaby** [comment]
这些应该都不考虑在 Core 中实现，我们暂时也没有开发插件的计划。

**Comment by ruibaby** [comment]
我将把此 issue 转移到 Discussion 的 Ideas 分区。
